src/pages/BlockDetail/BlockComp.tsx

```diff
@@ -164,16 +166,16 @@ const BlockOverview = ({ block }: { block: State.Block }) => {
       ),
     },
     {
-      title: i18n.t('block.difficulty'),
-      content: localeNumberString(block.difficulty),
+      title: i18n.t('block. difficulty'),
```

Comment:
Why insert a blank in the word?

Reply:
Oh. It is my default.

Reply:
Updated. https://github.com/nervosnetwork/ckb-explorer-frontend/pull/417/commits/af3fa49fe58db3bde1872e541d5049e4c4d6acef